packages/paste-website/src/pages/components/side-modal/index.mdx

```diff
@@ -0,0 +1,364 @@
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/side-modal"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(frontmatter: {slug: {eq: "/components/side-modal/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+        description
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    allAirtable(filter: {data: {Feature: {eq: "Side Modal"}}}) {
+      edges {
+        node {
+          data {
+            Documentation
+            Figma
+            Design_committee_review
+            Engineer_committee_review
+            Code
+            status
+          }
+        }
+      }
+    }
+  }
+`;
+
+<NormalizedComponentHeader
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/side-modal"
+  storybookUrl="/?path=/story/components-side-modal--default"
+  description={props.data.mdx.frontmatter.description}
+  data={props.data}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+<LivePreview
+  scope={{
+    Paragraph,
+    SideModal,
+    SideModalBody,
+    SideModalButton,
+    SideModalContainer,
+    SideModalHeader,
+    SideModalHeading,
+  }}
+  noInline
+>
+  {defaultExample}
+</LivePreview>
+
+## Guidelines
+
+### About Side Modal
+
+The Side Modal component is a non-modal dialog that sits on top of the right side of the page. It is meant for situations like a preview of a record while looking at a table.
+
+<Callout variant="warning" marginY="space70">
+  <CalloutHeading as="h4">Only use one Side Modal on a page</CalloutHeading>
+  <CalloutText>
+    We currenly only support having one Side Modal open on a page. If you have questions, please post{' '}
+    <Anchor href="https://github.com/twilio-labs/paste/discussions" showExternal>
+      a GitHub Discussion
+    </Anchor>
+    .
+  </CalloutText>
+</Callout>
+
+### Accessibility
+
+Side Modal and non-modal dialogs follow these accessibility guidelines:
+
+- There must be a focusable element inside the dialog.
+- There should be a close button so screen readers have a specific close action to target.
+- A Side Modal is a focus trap, and focus is placed inside it when it's shown.
+- A Side Modal must be triggered by an explicit user action, e.g. clicking a button.
+
+## Examples
+
+### Basic Side Modal
+
+<LivePreview
+  scope={{
+    Paragraph,
+    SideModal,
+    SideModalBody,
+    SideModalButton,
+    SideModalContainer,
+    SideModalHeader,
+    SideModalHeading,
+  }}
+  noInline
+>
+  {defaultExample}
+</LivePreview>
+
+### Side Modal with a footer
+
+By default the footer actions are aligned to the right of the modal using the Flexbox property `justify-content`. This should be the most common configuration you should choose.
```

Comment:
suggestion: as it's a paired down API, you might consider just listing the alternative values and what they do here too.